### ISSUE TITLE
Fix compile warning

### ISSUE THREAD

**Issue by aseigo** [jshmrtn/email_checker#14]
Pretty trivial, not worth the hoops of a clone/push/pr.

Warning on compile on `lib/email_checker/check/smtp.ex:28` is:

    warning: function head for def valid?/2 must come at the top of its direct implementation.

Patch to fix:

```
diff --git a/lib/email_checker/check/smtp.ex b/lib/email_checker/check/smtp.ex
index 063fb34..914f92f 100644
--- a/lib/email_checker/check/smtp.ex
+++ b/lib/email_checker/check/smtp.ex
@@ -25,7 +25,6 @@ defmodule EmailChecker.Check.SMTP do
   """
   @spec valid?(String.t(), non_neg_integer) :: boolean
   def valid?(email, retries \\ max_retries())
-  def valid?(email, retries)
   def valid?(_, 0), do: false

   def valid?(email, retries) do
```

**Comment by maennchen** [comment]
Closed with #15

**Comment by aseigo** [comment]
thanks! :) :rocket:
